src/main/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchController.kt

```diff
@@ -240,8 +248,13 @@ class OffenderSearchController(
   fun findByTeamCode(
     @ApiParam(required = true, name = "teamCodeList")
     @PageableDefault pageable: Pageable,
-    @RequestBody teamCodeList: List<String>
+    @RequestBody @NotEmpty teamCodeList: List<String>
   ): SearchPagedResults {
     return searchService.findByListOfTeamCodes(pageable, teamCodeList)
   }
+
+  @ExceptionHandler(ConstraintViolationException::class)
```

Comment:
We have a `ControllerAdvice` class for collecting our common exception handlers so this should probably go in there